bower.json

```diff
@@ -41,7 +41,7 @@
     "purescript-nonempty": "master",
     "purescript-partial": "master",
     "purescript-prelude": "master",
-    "purescript-record": "master",
+    "purescript-record": "kl0tl/purescript-record#no-monomorphic-proxies",
```

Comment:
I’ll point back to master once https://github.com/purescript/purescript-record/pull/67 is merged.